src/pages/partner/index.md

```diff
@@ -99,7 +98,7 @@ Here’s your guide to onboarding on the Adobe I/O console and accessing the Glo
 
 The new OAuth Server-to-Server credentials rely on the OAuth 2.0 client credentials grant. Therefore, you can use industry-standard OAuth 2.0 libraries to implement access token generation in your application.
```

Comment:
We should just say:

OAuth Server-to-Server credentials rely on the OAuth 2.0 client credentials grant. Therefore, you can use industry-standard OAuth 2.0 libraries to implement access token generation in your application.

While OAuth Server-to-Server credentials do not use expiring certificates, they still allow client secret rotation through the UI and API.

That's it.

Reply:
done